doc/index.md

```diff
@@ -0,0 +1,74 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.13.8
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+Welcome to autotag-metadata's documentation!
+===================================
+
+Autotag-metadata allows automatic creation of metadata files upon file creation. The input metadata file can be added in a simple GUI or from template YAML files.
+The program was originally developed to create metadata files for experimental data acquired in a laboratory. However, it can in principle be used in any process where files are generated in the file system.
```

Comment:
```suggestion
The program was originally developed to create metadata files for experimental data acquired in a laboratory. However, it can be used in any process where files are generated in the file system and the attachment of additional metadata is desired.
```